recipes/igwn-lldd-common/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "igwn-lldd-common" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://software.igwn.org/lscsoft/source/igwn-lldd-common-0.1.0.tar.gz
```

It's probably best to use the jinja2 variables here:

```suggestion
  url: https://software.igwn.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz
```